packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -322,61 +314,54 @@ skipInNode(describe)('plugin-meetings', () => {
         assert.equal(analysis4.valid, true);
       });
 
-      it('alice Audio Mute ', () => Promise.all([
-        testUtils.delayedPromise(alice.meeting.muteAudio()),
-        testUtils.waitForEvents([{scope: bob.meeting.members, event: 'members:update'}])
-          .then((response) => {
-            const aliceParticipant = response[0].result.delta.updated.find((member) => alice.meeting.members.selfId === member.id);
+      it('alice Audio Mute ', () => {
+        const checkEvent = (event) => !!event.delta.updated.find((member) => alice.meeting.members.selfId === member.id && member.isAudioMuted === true);
 
-            console.log('Alice participant', JSON.stringify(aliceParticipant));
-            console.log('SELF STATE ', JSON.stringify(alice.meeting.locusInfo.participants));
-            assert.equal(aliceParticipant.isAudioMuted, true);
-          })
-      ])
-        .then(() => {
-          assert.equal(alice.meeting.audio.muted, true);
 
-          return testUtils.waitUntil(4000);
-        }));
+        return Promise.all([
+          testUtils.delayedPromise(alice.meeting.muteAudio()),
+          testUtils.waitForEvents([{scope: bob.meeting.members, event: 'members:update', match: checkEvent}])
```

Comment:
Above you moved the `waitForEvents` out of this `Promise.all`. Should you do the same thing for the following tests as well?